### PR TITLE
patient create by user, Redux code refractoring

### PR DESCRIPTION
**Pull request by aman007shrestha**

### CODE REVIEW

**Review by aayush-musyaju — APPROVED** [review]
Everything looks good to me!!

File: src/features/authentication/authSlice.tsx [hunk 1]
```diff
@@ -2,35 +2,78 @@ import { createSlice } from '@reduxjs/toolkit';
 import { PayloadAction } from '@reduxjs/toolkit';
 
 interface UserDetails {
-  userName: string;
+  username: string;
   isLoggedIn: boolean;
+  isAdmin: boolean;
```
Comment (crazieeKinG) [inline src/features/authentication/authSlice.tsx:7]: In user story, I think only manager need login system. So, I think isLoggedin attribute can act as isAdmin as well.

**Review by crazieeKinG — APPROVED** [review]
Everything looks good.

File: src/features/authentication/authSlice.tsx (hunk 1 above)
Comment (aman007shrestha) [reply src/features/authentication/authSlice.tsx:7]: I am not quite sure as of yet. There is also mentioned registered user story. I intend to make a login interface for users as well.

**Pull request merged**